RibonDAO/app
# [PROD3-173] New layout to give tickets to integration

# Description

-  New layout to give tickets to integration
- Adjust events on app download page

## Does this pull request close any open issues?

- Put here the issue that must be closed with this PR
- eg: closes [#PROD3-173](https://ribon.atlassian.net/jira/software/c/projects/PROD3/boards/28?selectedIssue=PROD3-173)

## Type of change

Please delete options that are not relevant.


## How to test

Put here all the things need to test this PR and verify your changes, also list any relevant details for tests (eg: have a wallet)

- Use a integration id on link and verify new copies and layout
